sdk/identity/azure-identity/setup.py

```diff
@@ -76,6 +76,7 @@
         "msal<2.0.0,>=1.0.0",
         "msal-extensions~=0.1.3",
         "six>=1.6",
+        "pywin32;platform_system=='Windows'",
```

Comment:
Given our history with it (#8292, #8573), I prefer not to expand our dependency on this library. We're using only one native API, is it reasonable to invoke it through `ctypes`?

Reply:
It is far more than one native API. I implemented with ctypes and by switching to pywin32, I cut more than 2/3 of my code. Given pywin32 is maintained by Microsoft, I don't see concerns to use it.

Reply:
[pywin32](https://github.com/mhammond/pywin32) isn't a Microsoft project and we've twice been broken by its releases. The second break was particularly bad: installing the package caused the interpreter to crash ever after. You couldn't even `pip uninstall` it. So, I'm happy to have more code in our library instead.

Reply:
Then how about [cffi](https://pypi.org/project/cffi/)?